dotcom-rendering/src/components/InteractivesDisableArticleSwipe.importable.tsx

```diff
@@ -3,12 +3,31 @@ import { useEffect } from 'react';
 import { getInteractionClient } from '../lib/bridgetApi';
 
 export const InteractivesDisableArticleSwipe = () => {
-	useEffect(() => {
-		void getInteractionClient()
+	const onTouchStart = () => {
+		getInteractionClient()
 			.disableArticleSwipe(true)
 			.catch((error) => {
-				log('dotcom', 'disableArticleSwipe failed:', error);
+				log('dotcom', 'disableArticleSwipe(true) failed', error);
+			});
+	};
+
+	const onTouchEnd = () => {
+		getInteractionClient()
+			.disableArticleSwipe(false)
+			.catch((error) => {
+				log('dotcom', 'disableArticleSwipe(false) failed', error);
 			});
+	};
```

Comment:
I'm wondering if you want the article to always have the article swipe disabled? If that's the case, it might be better to not ever set this to `true`, what do you think?

Reply:
Ah that's a great shout! Yes, we always want interactives to have the article swipe disabled, have removed `touchend` events with bd01f1cfbb0ea966d89c1c4d181d7d29f30738d8